api/v1/helpers/verbs/doGet.js

```diff
@@ -76,10 +76,20 @@ function doGet(req, res, next, props) {
     }
 
     getPromise.then((o) => {
+
+      // o is read only.
+      let returnObj = o;
+
+      // order collectors by name
+      if (props.modelName === 'Generator' && o.collectors) {
+        returnObj = JSON.parse(JSON.stringify(o.get()));
```

Comment:
I think you do not need to do stringify and then parse the JSON. just doing o.get is enough.

Reply:
the reason we are doing a deep copy is because o.get() returns a read only object. Hence we deep copy the o.get().